packages/craco/lib/features/webpack/merge-webpack-config.js

```diff
@@ -10,7 +10,7 @@ const { applyWebpackConfigPlugins } = require("../plugins");
 const {
     addPlugins: addWebpackPlugins,
     removePlugins: removeWebpackPlugins,
-    pluginByName
+    pluginByName,
```

Comment:
```suggestion
    pluginByName
```

Remove unnecessary autoformat.